src/lib/viewers/image/__tests__/ImageViewer-test.js

```diff
@@ -88,6 +89,7 @@ describe('lib/viewers/image/ImageViewer', () => {
         test('should set up layout', () => {
             expect(image.wrapperEl).toHaveClass('bp-image');
             expect(image.imageEl).toHaveClass('bp-is-invisible');
+            expect(image.imageEl.getAttribute('alt')).toBe('preview image tales');
```

Comment:
Since "preview" can be a verb, this reads almost as an action, like "click button submit." The "image" portion should be covered by the element's `img` tag, as well.